01_values.md

```diff
@@ -49,7 +49,7 @@ La memoria de la computadora solía ser mucho más pequeña, y la gente solía u
 
 {{index signo, "número de punto flotante", "bit de signo"}}
 
-Sin embargo, no todos los números enteros menores que 18 mil trillones encajan en un número de JavaScript. Esos bits también almacenan números negativos, por lo que un bit indica el signo del número. Un problema más grande es representar números no enteros. Para hacer esto, algunos de los bits se utilizan para almacenar la posición del punto decimal. El número entero máximo real que se puede almacenar está más en el rango de 9 cuatrillones (15 ceros), que sigue siendo increíblemente grande.
+Sin embargo, no todos los números enteros menores que 18 mil trillones encajan en un número de JavaScript. Esos bits también almacenan números negativos, por lo que un bit indica el signo del número. Un problema más grande es representar números no enteros. Para hacer esto, algunos de los bits se utilizan para almacenar la posición del punto decimal. El número entero máximo real que se puede almacenar está más en el rango de 9 mil billones (15 ceros), que sigue siendo increíblemente grande.
```

Comment:
I think it's better use the spanish version because cuatrillón in spanish comes from french and it's 10^24 not 10^15
[RAE - cuatrillón](https://www.rae.es/dpd/cuatrill%C3%B3n)